src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,20 @@
 
 public class MainApp {
 
+    public  static void main(String[] args) {
+
+        Truck truck =  new Truck();
+        Bulldozer bulldozer  = new Bulldozer();
+        Excavator excavator = new Excavator();
+
+        truck.doWork();
+        truck.stopWork();
+
+        bulldozer.doWork();
+        bulldozer.stopWork();
+
+        excavator.doWork();
+        excavator.stopWork();
```

Comment:
here better to create an array of Machines with Truck, Bulldozer and Excavator and call methods doWork and stopWork in a loop